.github/workflows/delete.yaml

```diff
@@ -15,6 +15,6 @@ jobs:
         run: |
           BRANCH=$(echo -n ${BRANCH} | tr -c '[:alnum:]._-' '-')
           TOKEN=$(curl -s -H "Content-Type: application/json" -X POST -d '{"username": "'${DOCKER_USER}'", "password": "'${DOCKER_PASS}'"}' https://hub.docker.com/v2/users/login/ | jq -r .token)
-          images=("${BRANCH}-latest" "${BRANCH}-runtime" "${BRANCH}-tools" "${BRANCH}-tests-1.25" "${BRANCH}-tests-1.24" "${BRANCH}-tests-1.23" "${BRANCH}-tests-1.22" "${BRANCH}-builder")
+          images=("${BRANCH}-latest" "${BRANCH}-runtime" "${BRANCH}-tools" "${BRANCH}-tests-1.26" "${BRANCH}-tests-1.25" "${BRANCH}-tests-1.24" "${BRANCH}-tests-1.23" "${BRANCH}-tests-1.22" "${BRANCH}-builder")
```

Comment:
We can probably use the opportunity and get rid of 1.22; and while we're here remove 1.23 as well since it's already unsupported upstream, and DOKS will follow soonish.

(If this is not part of the documentation, could I ask you to add it, i.e., suggest to use the opportunity when we add support for a new version to also consider removing unsupported versions?)

Reply:
Decided out-of-band that this can be handled by a follow-up PR.